app/src/index.js

```diff
@@ -1,17 +1,16 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
-import App from "./App";
-import reportWebVitals from "./reportWebVitals";
+import { ChakraProvider } from "@chakra-ui/react";
+
+import App from "../src/App.js";
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <ChakraProvider></ChakraProvider>
```

Comment:
If you are putting your signup component into App, it probably should be App here instead of <ChakraProvider>. I'm not sure what it's for, but this might be why the signup component is not rendering...